sources/packages/backend/apps/api/src/route-controllers/restriction/restriction.aest.controller.ts

```diff
@@ -278,13 +279,13 @@ export class RestrictionAESTController extends BaseController {
 
   /**
    * Get restrictions for an institution.
-   * @param institutionId id of the institution to retrieve its restrictions.
-   * @returns Institution Restrictions.
+   * @param institutionId ID of the institution to retrieve its restrictions.
+   * @returns Institution restrictions.
    */
   @Get("institution/:institutionId")
   async getInstitutionRestrictions(
     @Param("institutionId", ParseIntPipe) institutionId: number,
-  ): Promise<RestrictionSummaryAPIOutDTO[]> {
+  ): Promise<InstitutionRestrictionSummaryAPIOutDTO[]> {
```

Comment:
Can we add wrapping the array of restrictions within an Object as an item to future refactor list?

Reply:
I see the point, but right now it is keeping the behavior from before, and aligned with the student.
Unless there is a strong reason to be a blocker, I would keep it as it is.